app/views/content/is-teaching-right-for-me/computing/_article.html.erb

```diff
@@ -0,0 +1,137 @@
+<div class="row inset">
+  <section class="col col-720">
+    <p>
+      Teaching computing is an important role to equip pupils with knowledge and skills that will help them succeed in a  world dependent on digital technology.</p>
+
+<p>You'll inspire pupils to access new fields and careers like quantum computing and artificial intelligence. You could inspire the next generation of computer scientists. </p>
```

Comment:
@caitlin-reidy Computer scientists is quite broad, is it possible to give more specific examples of jobs people can relate to?

Reply:
artificial intelligence (AI)